include/etl/impl/cudnn/conv.hpp

```diff
@@ -12,6 +12,8 @@
 
 #pragma once
 
+#define ETL_TENSOR_CORES
```

Comment:
Could you rename this into `ETL_EXPERIMENTAL_TENSOR_CORES`? This would reflect better the state of this feature.